src/libcmd/common-eval-args.cc

```diff
@@ -34,10 +34,13 @@ MixEvalArgs::MixEvalArgs()
         .shortName = 'I',
         .description = R"(
   Add *path* to the Nix search path. The Nix search path is
-  initialized from the colon-separated [`NIX_PATH`](./env-common.md#env-NIX_PATH) environment
-  variable, and is used to look up the location of Nix expressions using [paths](../language/values.md#type-path) enclosed in angle
+  initialized from the colon-separated [`NIX_PATH`] environment
+  variable, and is used to look up the location of Nix expressions using [paths] enclosed in angle
   brackets (i.e., `<nixpkgs>`).
 
+  [`NIX_PATH`]: ../env-common.md#env-NIX_PATH
+  [paths] ../../language/values.md#type-path
```

Comment:
```suggestion
  [paths]: ../../language/values.md#type-path
```

Reply:
BTW what is the advantage of using footnotes over direct links? Seems like more opportunity to make a typo.

Reply:
It's not always clearly better, but usually makes the prose part of the source much more readable and makes it slightly easier to update URLs, especially for bulk changes. I suppose we agree that markdown has no optimal answers.